assets/crd/route.crd.yaml

```diff
@@ -12,381 +12,287 @@ spec:
+                      type: string
+                    termination:
+                      description: "termination indicates termination type. \n * edge - TLS termination is done by the router and http is used to communicate with the backend (default) * passthrough - Traffic is sent straight to the destination without the router providing TLS termination * reencrypt - TLS termination is done by the router and https is used to communicate with the backend"
                       enum:
-                      - Subdomain
-            description: spec is the desired state of the route
-            properties:
-              alternateBackends:
-                description: alternateBackends allows up to 3 additional backends
-                  to be assigned to the route. Only the Service kind is allowed, and
-                  it will be defaulted to Service. Use the weight field in RouteTargetReference
-                  object to specify relative preference.
-                items:
-                  description: RouteTargetReference specifies the target that resolve
-                    into endpoints. Only the 'Service' kind is allowed. Use 'weight'
-                    field to emphasize one over others.
+                        - edge
+                        - reencrypt
+                        - passthrough
+                      type: string
+                  required:
+                    - termination
+                  type: object
+                to:
+                  description: to is an object the route should use as the primary backend. Only the Service kind is allowed, and it will be defaulted to Service. If the weight field (0-256 default 100) is set to zero, no traffic will be sent to this backend.
                   properties:
                     kind:
                       default: Service
-                      description: The kind of target that the route is referring
-                        to. Currently, only 'Service' is allowed
+                      description: The kind of target that the route is referring to. Currently, only 'Service' is allowed
                       enum:
-                      - Service
-                      - ""
+                        - Service
+                        - ""
                       type: string
                     name:
-                      description: name of the service/target that is being referred
-                        to. e.g. name of the service
+                      description: name of the service/target that is being referred to. e.g. name of the service
                       minLength: 1
                       type: string
                     weight:
-                      description: weight as an integer between 0 and 256, default
-                        100, that specifies the target's relative weight against other
-                        target reference objects. 0 suppresses requests to this backend.
+                      default: 100
```

Comment:
The schema diff looks scary due to https://github.com/openshift/api/commit/c53e08e0561c14644db0f22b5b4e3574ea746e2b, but this default value for "weight" is the thing that is relevant to route/v1 API compatibility.